keras_cv/bounding_box/to_ragged_test.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+from keras_cv import bounding_box
+
+
+class ToRaggedTest(tf.test.TestCase):
+    def test_converts_to_ragged(self):
```

Comment:
Maybe a round-trip test with randomly initialized boxes would be good?

Reply:
Done